app/views/jobseekers/job_applications/build/qualifications.html.slim

```diff
@@ -33,8 +33,8 @@
                                     tag.div(qualification.finished_studying_details.presence, class: "govuk-body")]))
 
           - detail.actions do
-            = govuk_button_to t("buttons.delete"), jobseekers_job_application_qualification_path(job_application, qualification), method: :delete, class: "govuk-delete-link govuk-!-margin-bottom-0", form_class: "inline-block button_to"
-            = govuk_link_to t("buttons.change"), edit_jobseekers_job_application_qualification_path(job_application, qualification), class: "govuk-link--no-visited-state inline-block"
+            = govuk_link_to t("buttons.change"), edit_jobseekers_job_application_qualification_path(job_application, qualification), class: "govuk-link--no-visited-state"
+            = govuk_link_to t("buttons.delete"), jobseekers_job_application_qualification_path(job_application, qualification), method: :delete
```

Comment:
Why are you changing this in this PR?

Reply:
to be consistent and have it in the right order and with the right style, it should not be a button